runtime/src/main/java/org/corfudb/runtime/collections/TxnStreamingManager.java

```diff
@@ -41,6 +41,12 @@
      */
     private final Map<Integer, StreamingSubscriptionContext> subscriptions = new HashMap<>();
 
+    /**
+     * TransactionPoller context to pull updates from the streams.
+     * There should be only 1 for all subscribers.
+     */
+    TransactionPoller transactionPoller;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4695667)